### ISSUE TITLE
Project stuck loading only in Test Mode

### ISSUE THREAD

**Issue by ReCoL1337** [FlutterFlow/flutterflow-issues#1763]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

Currently my project works in Run mode, but not in Test mode. Test mode never finishes "loading"

![image](https://github.com/FlutterFlow/flutterflow-issues/assets/46446137/d4aad4f5-d57e-4c6c-93e6-db9291582721)

### Expected Behavior

Test mode should finish loading so I can test any changes I make and reload the session.

### Steps to Reproduce

Open Test Mode on my project: https://app.flutterflow.io/project/random-web-l93wej

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4ghvLlvJROxL1E0M/ucO9Vu2U4Qm4jaL00sNobbxAjIoDoPJkUaczCXBJuUuardG9tKEWKml8B+Pz7vYXAUvBfNTisbbZDyJZxFzyUfHq/aoSwOqWORHNQHcFQf0Cw0KbZkxIkNbJ1LF4R3Ga6B/K/TXXaGJv7Zwh9f6PHaOI=

### Context

Okay so this is basically a copy of: https://github.com/FlutterFlow/flutterflow-issues/issues/1660 because @hariprasadms told me to open another ticket. This was a sudden error - I was working on my project normally one day, closed everything and then when I re-opened it Test Mode just wasn't working. I think there is something with the CanvasKit renderer because when I created a custom widget that needed the CanvasKit render in order to work, it wasn't working in test mode for some reason (it gave me errors that I don't have CanvasKit enabled). When I opened it in the run mode and on the hosting everything worked fine. I don't recall exactly when this happened but shortly after that my test mode wasn't working at all (returning the error above). And yes, I have enabled the renderer in the settings. Not sure that this is the reason but I thought I could include it. As this is only my second project I haven't experienced this error in other projects. I'm currently using FlutterFlow 4.0 and Flutter 3.13.7. There's no error indicated by these versions. Also tried it on different browsers: Opera GX, Google Chrome, Firefox.
![image](https://github.com/FlutterFlow/flutterflow-issues/assets/46446137/3184008c-d885-4dc5-9a20-1300baf8ff2a)

### Visual documentation

![image](https://github.com/FlutterFlow/flutterflow-issues/assets/46446137/d4aad4f5-d57e-4c6c-93e6-db9291582721)
![image](https://github.com/FlutterFlow/flutterflow-issues/assets/46446137/bae323b7-803b-4938-8067-136cb5256b18)

### Environment

```markdown
- FlutterFlow version: v4.0
- Platform: Web
- Browser name and version: Opera GX, Google Chrome, Firefox
- Operating system and version affected: Windows 11
```

**Comment by hariprasadms** [comment]
Hi @ReCoL1337 - Thanks for submitting the issue. I appreciate the details you provide. This issue is more specific to project.

**Comment by katherineqian** [comment]
Hi @ReCoL1337, sorry for the delay here. I've taken a look at your project and identified the problem as a duplicate of https://github.com/FlutterFlow/flutterflow-issues/issues/1795. The issue with Test Mode is being caused by the import of the csv package in your custom actions. I don't have a solution for you yet as we are unsure what's causing this problem in Test Mode only, but I wanted to provide that update in case it can help you be unblocked and work on other parts of your project in the meantime. I'm going to close this issue so that we can track them all together -- please follow that issue for further updates (feel free to leave a comment and future replies should send you an email notification). Thank you!
